client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -843,9 +849,26 @@ export const ApplicationsTable: React.FC = () => {
                 {getColumnVisibility("tags") && (
                   <Th {...getThProps({ columnKey: "tags" })} width={10} />
                 )}
-                {getColumnVisibility("effort") && (
-                  <Th {...getThProps({ columnKey: "effort" })} width={10} />
-                )}
+                <Th {...getThProps({ columnKey: "effort" })}>
+                  <Flex
+                    flexWrap={{ default: "nowrap" }}
+                    spaceItems={{ default: "spaceItemsSm" }}
+                  >
+                    <FlexItem>
+                      <span>{t("effort")}</span>
```

Comment:
This should be `terms.effort` since the `effort` key is located under the "terms" group.

With i18next, if the key doesn't exist in the translations.json, the library will use the key's name instead. The results will be a lower case "effort" and that may be breaking the e2e testing.  See https://github.com/konveyor/tackle2-ui/actions/runs/8803551797/job/24162609546#step:10:1817